terraform/layers/base/vpc.tf

```diff
@@ -0,0 +1,22 @@
+data "aws_availability_zones" "available_azs" {
+  state = "available"
+}
+
+locals {
+  az_names = slice(data.aws_availability_zones.available_azs.names, 0, 3)
+}
+
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  name = var.lab-1-vpc-name
```

Comment:
This approach is fine but might be worth naming resources with project and environment. E.g. 
`name = "${var.project_name}-${var.environment}-vpc"` 

or creating a local, an example would be 
```
locals {
  resource_name_prefix = "${var.project_name}-${var.environment}"
}
```

and naming resources like `name = "${local.resource_name_prefix}-vpc"`

Might help down the line - common approaches to help quickly see what reosources belong to which project / environment

Reply:
Makes sense, thank you!